quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/approvals/cli/help/HelpCommandTest.testStartDevHelp.unix.approved.txt

```diff
@@ -46,6 +46,10 @@ Hostname:
 
 --hostname <hostname>
                      Hostname for the Keycloak server.
+--hostname-admin <hostname>
```

Comment:
could you also add this to the windows based approval files pls? I know it is not very convenient atm, but without it the testsuite would break on windows ;)